### ISSUE TITLE
LiveSync on iOS simulator throws error

### ISSUE THREAD

**Issue by NickBranstein** [NativeScript/nativescript-cli#824]
When running the following command I receive an error.

```
tns livesync ios --emulator --watch
```

### Error
```
Command failed: /bin/sh -c xcrun simctl launch E61F7463-84D6-4214-BE7A-14F1E57328E3 org.nativescript.lab2finish
An error was encountered processing the command (code=146):
Exception encountered connecting to CoreSimulatorBridge: Unable to connect to CoreSimulatorBridge
```

### Additional Info
The simulator does launch even after I get this error message however the livesync functionality does not work.

* I'm using CLI Version 1.2.4
* I have tried this on multiple Summer of NativeScript labs - most recently Lab 2
* After cloning the Summer of NativeScript repo I ran tns platform update ios
* iOS Simulator is version 8.4
* XCode 6.4
* OS X 10.10.4

### Additional Trace
https://dl.dropboxusercontent.com/u/34791521/additionalTrace.log

**Comment by Fatme** [comment]
Hi @NickBranstein,

It seems that your application is not deployed on the simulator. Try to execute `tns run ios --emulator` before `livesync` command.

**Comment by NickBranstein** [comment]
Thanks @Fatme.  It's working for me now.

I had closed my simulator prior to running livesync (after my original run).  Even though the app was still deployed it seems that I have to leave the simulator open after running the first time.  Is this intended - I guess assumed livesync was going to relaunch my simulator if it was closed (which is does but that error comes up before the simulator starts).
